README.md

```diff
@@ -9,9 +11,35 @@ Full featured clan system for PVP Minecraft Servers!
 [![Crowdin](https://badges.crowdin.net/simpleclans/localized.svg)](https://crowdin.com/project/simpleclans)
 [![Discord](https://img.shields.io/discord/719557355917934613?label=discord&logo=discord)](https://discord.gg/CkNwgdE)
 
-Spigot resource page: https://www.spigotmc.org/resources/simpleclans.71242/
+#### Download  
+
+* [Spigot Resource](https://www.spigotmc.org/resources/simpleclans.71242/)
+* [GitHub Releases](https://github.com/RoinujNosde/SimpleClans/releases)
```

Comment:
This does not contain download links for newer releases.

Reply:
So you want it to link to https://github.com/RoinujNosde/SimpleClans/releases/latest ?

Reply:
No need to link to GitHub releases. Releases contains only changelogs. The downloads are on Spigot (and Bintray, for devs).

Reply:
oh right! ok now I understand. Let me remove it.

Reply:
Ive updated it on my fork, I have no idea what I need to do to update it here